moped-editor/src/views/projects/projectView/ProjectComponentsMap.js

```diff
@@ -223,29 +225,33 @@ const ProjectComponentsMap = ({
   drawLines = null,
 }) => {
   const classes = useStyles();
-  const selectedLayerIds = createSelectedIdsObjectFromFeatureCollection(
-    featureCollection
-  );
+  const selectedLayerIds =
+    createSelectedIdsObjectFromFeatureCollection(featureCollection);
   const mapRef = useRef();
-  const mapGeocoderContainerRef = useRef();
   const mapEditToolsContainerRef = useRef();
   const mapBasemapContainerRef = useRef();
   /**
    * Generate a viewport configuration object
    */
-  const [viewport, setViewport] = useFeatureCollectionToFitBounds(
+
+  const [viewport, setViewport] = useState(mapConfig.mapInit);
+  /**
+   * Make use of a custom hook that initializes a map viewport
+   * and fits it to a provided feature collection.
+   */
+  const { fitMapToFeatureCollectionOnRender } = useFeatureCollectionToFitBounds(
     mapRef,
     // If this is a new feature, use the project feature collection to retrieve the area
     newFeature ? projectFeatureCollection : featureCollection,
     isSignalComponent
   );
 
-  const {
-    handleLayerHover,
-    featureText,
-    featureId,
-    hoveredCoords,
-  } = useHoverLayer();
+  const [cursor, setCursor] = useState("auto");
+  const onMouseEnter = useCallback(() => setCursor("pointer"), []);
+  const onMouseLeave = useCallback(() => setCursor("auto"), []);
```

Comment:
Cursor updates are handled differently in v7 and I used [this example from the docs](https://github.com/visgl/react-map-gl/tree/7.0-release/examples/custom-cursor)

Reply:
v nice!